[2024-04-27 23:19:33,856] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: exampleSimon.runme_2 2024-04-26T00:00:00+00:00 [queued]>
[2024-04-27 23:19:33,940] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: exampleSimon.runme_2 2024-04-26T00:00:00+00:00 [queued]>
[2024-04-27 23:19:33,943] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2024-04-27 23:19:33,946] {taskinstance.py:1069} INFO - Starting attempt 1 of 1
[2024-04-27 23:19:33,948] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2024-04-27 23:19:34,055] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): runme_2> on 2024-04-26T00:00:00+00:00
[2024-04-27 23:19:34,111] {standard_task_runner.py:52} INFO - Started process 221 to run task
[2024-04-27 23:19:34,220] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'exampleSimon', 'runme_2', '2024-04-26T00:00:00+00:00', '--job-id', '90', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/helloAgain.py', '--cfg-path', '/tmp/tmp_7g18g3_', '--error-file', '/tmp/tmpsgrqs9_l']
[2024-04-27 23:19:34,900] {standard_task_runner.py:77} INFO - Job 90: Subtask runme_2
[2024-04-27 23:19:51,400] {logging_mixin.py:104} INFO - Running <TaskInstance: exampleSimon.runme_2 2024-04-26T00:00:00+00:00 [running]> on host 10d8235a57dc
[2024-04-27 23:19:52,070] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=exampleSimon
AIRFLOW_CTX_TASK_ID=runme_2
AIRFLOW_CTX_EXECUTION_DATE=2024-04-26T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-04-26T00:00:00+00:00
[2024-04-27 23:19:52,111] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-04-27 23:19:52,120] {bash.py:158} INFO - Running command: echo "exampleSimon__runme_2__20240426" && sleep 1
[2024-04-27 23:19:52,299] {bash.py:169} INFO - Output:
[2024-04-27 23:19:52,357] {bash.py:173} INFO - exampleSimon__runme_2__20240426
[2024-04-27 23:19:53,368] {bash.py:177} INFO - Command exited with return code 0
[2024-04-27 23:19:53,493] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=exampleSimon, task_id=runme_2, execution_date=20240426T000000, start_date=20240427T231933, end_date=20240427T231953
[2024-04-27 23:19:53,672] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-27 23:19:53,751] {local_task_job.py:146} INFO - Task exited with return code 0
